gcp/modules/couchdb/main.tf

```diff
@@ -3,13 +3,47 @@ terraform {
 }
 
 variable "values_dir" {}
+variable "release_namespace" {
+	default = "gpii"
+}
 
 module "couchdb" {
   source = "/exekube-modules/helm-template-release"
 
   release_name      = "couchdb"
-  release_namespace = "gpii"
+  release_namespace = "${var.release_namespace}"
   release_values    = "${var.values_dir}/couchdb.yaml"
 
   chart_name = "../../../../../charts/couchdb"
 }
+
+variable "couchdb_admin_username" {}
+variable "couchdb_admin_password" {}
+
+resource "null_resource" "couchdb_finish_cluster" {
+  depends_on = ["module.couchdb"]
+
+  provisioner "local-exec" {
+    command = <<EOF
+      while [ "$PODS_READY" != "true" ]; do
```

Comment:
I would use lower case for all these local vars, i.e. `$pods_ready` rather than `$PODS_READY`. Note that you already use `$status` and not `$STATUS`.

Reply:
I would prefer to keep upper case everywhere instead, it really makes it more readable. I will commit new version after my tests are finished, please check if it looks better.